audience_manager/index.rst

```diff
@@ -0,0 +1,14 @@
+.. PPMS-PDFD documentation master file, created by
+   sphinx-quickstart on Tue Dec 12 14:22:39 2017.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Welcome to Piwik PRO Audience Manager documentation!
+====================================================
+
+.. toctree::
+  :maxdepth: 2
```

Comment:
This value will control how deep will be ToC and displayed sidebar links. You should check how they look after generation and adjust it accordingly.